src/tutorial/counter-app/event.md

```diff
@@ -39,9 +41,19 @@ use `mpsc` to set us up for the future.
 ```
 
 Finally, here's the code that starts a thread that polls for events from `crossterm` and maps it to
-our `Event` enum.
+our `Event` enum. 
 
 ```rust
+use std::{
+    sync::mpsc,
+    thread,
+    time::{Duration, Instant},
+};
+
+use crossterm::event::{self, Event as CrosstermEvent, KeyEvent, MouseEvent};
```

Comment:
If you modify `src/event.rs` for the `ratatui-counter-app` like so:

```
///// ANCHOR: event_import
use std::{
  sync::mpsc,
  thread,
  time::{Duration, Instant},
};

use anyhow::Result;
use crossterm::event::{self, Event as CrosstermEvent, KeyEvent, MouseEvent};
///// ANCHOR: event_import
```

then this block can become:

```suggestion
{{#include ./ratatui-counter-app/src/event.rs:event_import}}
```